backend/src/test/java/pl/edu/agh/awiteks_backend/utilities/UserDataValidationUtilitiesTest.java

```diff
@@ -0,0 +1,93 @@
+package pl.edu.agh.awiteks_backend.utilities;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class UserDataValidationUtilitiesTest {
+
+    @Autowired
+    private UserDataValidationUtilities userDataValidationUtilities;
+
+    // PASSWORD TESTS
+
+    @Test
+    @DisplayName("Putting good password test")
+    void properPassword() {
+        testPassword("1Ab.;'@#ls", true);
+    }
+
+    @Test
+    @DisplayName("Putting password with no small chars test")
+    void noSmallCharsPassword() {
+        testPassword("1AAABBB23", false);
+    }
+
+    @Test
+    @DisplayName("Putting password with no big chars test")
+    void noBigCharsPassword() {
+        testPassword("aaaabbbb32435", false);
+    }
+
+    @Test
+    @DisplayName("Putting password with no numbers chars test")
+    void noNumbersPassword() {
+        testPassword("AAAbbbbbEEFs", false);
+    }
+
+    // USERNAME TESTS
```

Comment:
Pewnie wiesz i to absolutnie nie jest wymaganie poprawy, ale można użyć parametryzowanych testów:
https://www.baeldung.com/parameterized-tests-junit-5

Imo by to trochę upiększyło ten kod i pozwoliło na pozbycie się tych komentarzy i redundancji.

Reply:
Gdzieś kojarzyłem taką opcję, ale kompletnie nie wpadłem na pomysł, że można tego użyć. Dzięki wielkie za sugestię, zrefaktoryzowałem te testy.